bitrise.yml

```diff
@@ -1572,6 +1572,10 @@ workflows:
             sed 's/.*=//;s/'\''/"/g' data.txt > test.json
             cat test.json
             python3 perfTestTransform.py
+    meta:
+      bitrise.io:
+        stack: osx-xcode-15.4.x
```

Comment:
@clarmso would we be able to build using previous stack? I know we did this for L10nBuild workflow so trying same approach

Reply:
we can not test this workflow because the IPA is only generated from main, not from PRs

Reply:
No, the current change in `main` is not backwards compatible.

Reply:
the L10nBuil is building successfully with this approach 🤔  can we give this a try? This will only change the perf workflow

Reply:
No, the compilation failed but Bitrise's status is green. 🤔 
https://app.bitrise.io/build/8ad12ec5-3992-4b77-8ebc-01921b9ef345?tab=log

Reply:
we should fix that.. if that script step fails, the notification should be red not green :(